code/modules/food_and_drinks/food/snacks_other.dm

```diff
@@ -569,3 +579,12 @@
 	filling_color = "#A0522D"
 	tastes = list("chocolate" = 1)
 	foodtype = JUNKFOOD | SUGAR
+
+/obj/item/reagent_containers/food/snacks/riceball
```

Comment:
Why not just make a /food/snacks/sushi and make these all subtypes of that? You seem to be rewriting code here a number of times for no reason.

Reply:
Their like this to fit in with other food groups that of grains like rice pudding or such

Reply:
Use inheritance.